Dockerfile

```diff
@@ -140,6 +140,9 @@ RUN source /etc/default/evm && \
     yarn install && \
     gulp build
 
+## Clean the logs
+RUN rm -rf ${APP_ROOT}/log/*
```

Comment:
Just noticed ManageIQ/manageiq-appliance-build#138 and I commented there as well.